application/models/post.go

```diff
@@ -1007,3 +1015,28 @@ func (p *Post) Meeting() (*Meeting, error) {
 
 	return &meeting, nil
 }
+
+// FilterDestination returns a list of all posts with a Destination near the given location. The database is not
+// touched.
+func (p Posts) FilterDestination(location Location) Posts {
+	filtered := make(Posts, 0)
+	for i := range p {
+		_ = DB.Load(&p[i], "Destination")
```

Comment:
Can this go before the `for loop` and act on all the posts at once? (Same for next function)

Reply:
Good thought. It doesn't combine into one DB query, but a little simpler nonetheless.